particles/Tutorial/Kotlin/README.md

```diff
@@ -292,13 +300,33 @@ class DisplayGreetingParticle : AbstractDisplayGreetingParticle() {
 And finally, the BUILD file. Note the `arcs_kt_schema` rule which will generate schemas of the form
 '$ParticleName_EntityName'. This is why `DisplayGreeting.kt` referenced `GetPerson_Person()` while `DisplayGreeting.kt`
 referenced `DisplayGreeting_Person()`.
+
+On that note: since we decided to utilize one manifest file for shared schemas, we need to define that
+separately. By defining it in an `arcs_manifest`, we give ourselves the option to depend on these schemas
+in many particles down the line.
+
```

Comment:
How about:

On that note: since we put the Person schema in its own Arcs Manifest file, we need to include this in our BUILD rule. To do this, we define an `arcs_manifest` rule. This gives us the option to use these schemas in many particles and recipes down the line.

Reply:
Sure :+1: